pkg/controller/installation/products/amqstreams/reconciler.go

```diff
@@ -24,6 +24,7 @@ var (
 	defaultInstallationNamespace = "amq-streams"
 	defaultSubscriptionName      = "integreatly-amq-streams"
 	clusterName                  = "integreatly-cluster"
+	operatorVersion              = "1.1.0"
```

Comment:
I really think this should be controlled by 1 file which could be easily overridden by DeLorean

Reply:
discussed in another comment.